Dockerfile

```diff
@@ -0,0 +1,41 @@
+# AlpineLinux with a glibc-2.23, Oracle Java 8, sbt and git
+FROM anapsix/alpine-java:8_jdk
```

Comment:
thoughts on using Java 11 LTS instead of 8?

Reply:
No opinion - is it better in some way?

It's a little hard to get a docker image with a working oracle jdk in it, and someone already did 99% of the work for us with java 8 but not so with java 11.

Reply:
Let's leave it at 8 for now. I brought it up in the interest of pushing out the day when the language version will be out of date but getting up and running is higher priority right now.

Reply:
Did a little more reading - I think that openjdk works fine with sbt, so I'm experimenting.  openjdk is so much easier!  :)

Reply:
Great, all the tests still pass with it.  :)